bridge/slack/slack.go

```diff
@@ -311,6 +317,48 @@ func (b *Bslack) sendRTM(msg config.Message) (string, error) {
 	return b.postMessage(&msg, messageParameters, channelInfo)
 }
 
+func (b *Bslack) updateTopicOrPurpose(msg *config.Message, channelInfo *slack.Channel) (bool, error) {
+	var updateFunc func(channelID string, value string) (*slack.Channel, error)
+
+	incomingChangeType, text := extractTopicOrPurpose(msg.Text)
+	switch incomingChangeType {
+	case "topic":
+		updateFunc = b.rtm.SetTopicOfConversation
+	case "purpose":
+		updateFunc = b.rtm.SetPurposeOfConversation
+	default:
+		b.Log.Errorf("Unhandled type received from extractTopicOrPurpose: %s", incomingChangeType)
```

Comment:
You need to return after this or else the function will panic below on attempting to call a `nil`-pointer function.

Reply:
This is not test code. This is production code. So just an appropriate return would be great. :)

Reply:
haha oh dear. Ok, addressed in next push.

Say, is our process to let the person "raising" comment to mark as resolved, or is "marking as resolved" like a TODO list for me in this situation? I just realized I've been treating it like the latter, but not sure what our preference is...

cc: @42wim

Reply:
@patcon use it as your own TODO checklist. I'll verify anyway regardless as a reviewer and wil re-open where relevant.